### ISSUE TITLE
Elevated urinary homogentisic acid

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6717]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Alkaptonuria

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
